discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -22,9 +24,10 @@
 class LibvirtController(NodeController, ABC):
     TEST_DISKS_PREFIX = "ua-TestInfraDisk"
 
-    def __init__(self, private_ssh_key_path: Path):
+    def __init__(self, config: ControllerConfig, cluster_config: BaseClusterConfig):
```

Comment:
You changed the signature but some of usages left unchanged  e.g. `download_logs.py`